deploy-board/deploy_board/settings.py

```diff
@@ -301,6 +301,8 @@
 
     # Nimbus service url
     NIMBUS_SERVICE_URL = os.getenv("NIMBUS_SERVICE_URL", None)
+    NIMBUS_EGRESS_URL = os.getenv("NIMBUS_EGRESS_URL", None)
```

Comment:
can we just use "NIMBUS_SERVICE_URL" directly?

Reply:
@yongwen we also need the EGRESS_URL so we can send the request to localhost:19193 but set the host header with NIMBUS_SERVICE_URL.